newIDE/app/src/Profile/ContributionsDetails.js

```diff
@@ -0,0 +1,234 @@
+          <Trans>{`Extensions (${extensions.length})`}</Trans>
+        </Text>
+      </AccordionHeader>
+      <AccordionBody disableGutters>
+        <Column>
+          {extensions.map(extension => (
+            <ContributionLine
+              shortDescription={extension.shortDescription}
+              fullName={extension.fullName}
+              previewIconUrl={extension.previewIconUrl}
+            />
+          ))}
+        </Column>
+      </AccordionBody>
+    </Accordion>
+  ) : (
+    <PlaceholderLoader />
+  );
+};
+
+// Change examples to assets when the feature is developed.
+type AssetsAccordionProps = {|
+  examples: ?Array<ExampleShortHeader>,
+|};
+
+const AssetsAccordion = ({ examples }: AssetsAccordionProps) => {
+  return examples ? (
+    <Accordion>
+      <AccordionHeader>
+        <Text displayInlineAsSpan>
+          <Trans>Assets (coming soon!)</Trans>
+        </Text>
+      </AccordionHeader>
+    </Accordion>
+  ) : (
+    <PlaceholderLoader />
+  );
+};
+
+type Props = {|
+  userId: string,
+|};
+
+export default ({ userId }: Props) => {
+  const [
+    extensions,
+    setExtensions,
+  ] = React.useState<?Array<ExtensionShortHeader>>(null);
+  const [examples, setExamples] = React.useState<?Array<ExampleShortHeader>>(
+    null
+  );
+  const [extensionError, setExtensionError] = React.useState<?Error>(null);
+  const [exampleError, setExampleError] = React.useState<?Error>(null);
+
+  const fetchUserExtensions = React.useCallback(
+    () => {
+      (async () => {
+        if (!userId) return;
+        try {
+          const extensions = await getUserExtensionShortHeaders(userId);
+          setExtensions(extensions);
+        } catch (error) {
+          console.error('Error while loading extensions:', error);
+          setExtensionError(error);
+        }
+      })();
+    },
+    [userId]
+  );
+
+  React.useEffect(
+    () => {
+      fetchUserExtensions();
+    },
+    [fetchUserExtensions]
+  );
+
+  const fetchUserExamples = React.useCallback(
+    () => {
+      (async () => {
+        if (!userId) return;
+        try {
+          const examples = await getUserExampleShortHeaders(userId);
+          setExamples(examples);
+        } catch (error) {
+          console.error('Error while loading examples:', error);
+          setExampleError(error);
+        }
+      })();
+    },
+    [userId]
+  );
+
+  React.useEffect(
+    () => {
+      fetchUserExamples();
+    },
+    [fetchUserExamples]
+  );
+
+  return (
+    <>
+      {extensions && examples && (
+        <Column>
+          <Line alignItems="center">
+            <Text size="title">
+              <Trans>Contributions</Trans>
+            </Text>
+          </Line>
+        </Column>
+      )}
+      <ExtensionsAccordion
+        extensions={extensions}
+        extensionError={extensionError}
+      />
+      <ExamplesAccordion examples={examples} exampleError={exampleError} />
+      <AssetsAccordion examples={examples} />
+    </>
```

Comment:
Idea: we can maybe add a `BackgroundText` (once the loading is done) telling "Missing some contributions? Check that your username is added to each example and extensions you've made or contributed to."

Reply:
I like that, however what will the user do concretely if they realise they're not in the contributors? Open an issue? Send a message on Discord? 
We should maybe be more specific in the message.
How about:
"Missing some contributions? If you are the author, then create a Pull Request on the corresponding repository after adding your username in the authors (example or extension), otherwise ask the author to add you as a contributor!"

Reply:
Ok, this maybe then?

"Missing some contributions? If you are the author, create a Pull Request on the corresponding GitHub repository after adding your username in the authors of the example or the extension - or directly ask the original author to add your username."